op-service/metrics/http.go

```diff
@@ -44,21 +44,42 @@ func (n *noopHTTPRecorder) RecordHTTPRequest(*HTTPParams) {}
 func (n *noopHTTPRecorder) RecordHTTPResponse(*HTTPParams) {}
 
 type PromHTTPRecorder struct {
-	HTTPRequestDuration  *prometheus.HistogramVec
+	// HTTPRequestDuration is the old metric for request latency
+	// it was created with too tight buckets: [.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10]
+	// in order to preserve backward compatibility we are keeping this metric for now
+	// and it will be removed when services opt in to HTTPRequestLatency
+	// Deprecated: HTTPRequestDuration is deprecated
+	HTTPRequestDuration *prometheus.HistogramVec
+
+	// HTTPRequestLatency measures request execution latency in *seconds*
+	// buckets are: [.025, .05, .1, .25, .5, 1, 2.5, 5, 10, 25, 50]
```

Comment:
Didn't you want a 100 seconds bucket too?

Reply:
changed my mind - 50s is enough